File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/viewholders/TvSettingsViewHolder.java
// SPDX-License-Identifier: GPL-2.0-or-later

package org.dolphinemu.dolphinemu.viewholders;

import android.view.View;

import androidx.leanback.widget.ImageCardView;
import androidx.leanback.widget.Presenter;

public final class TvSettingsViewHolder extends Presenter.ViewHolder
{
  public ImageCardView cardParent;

  // Determines what action to take when this item is clicked.
  public int itemId;

  public TvSettingsViewHolder(View itemView)
  {
    super(itemView);

    itemView.setTag(this);

    cardParent = (ImageCardView) itemView;
  }
}

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/viewholders/TvSettingsViewHolder.kt
// SPDX-License-Identifier: GPL-2.0-or-later

package org.dolphinemu.dolphinemu.viewholders

import android.view.View
import androidx.leanback.widget.ImageCardView
import androidx.leanback.widget.Presenter

class TvSettingsViewHolder(itemView: View) : Presenter.ViewHolder(itemView) {
    var cardParent: ImageCardView

    // Determines what action to take when this item is clicked.
    var itemId = 0

    init {
        itemView.tag = this
        cardParent = itemView as ImageCardView
    }
}
